sophosendpointprotection.json

```diff
@@ -0,0 +1,2207 @@
+{
```

Comment:
Can we change indentation to 4 tabs as almost all the apps are following the same standard?

Reply:
yes updated indentation to 4 tabs.

Reply:
Sorry, for the confusion. I meant 4 spaces, instead of 2 spaces.